basiclibrary/src/test/java/basiclibrary/LibraryTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+//test feature1
+public class LibraryTest {
+    @Test public void testRollMethod(){
+        int number=4;
+        int[]result = Library.roll(number);
+        assertEquals(number,result.length);
+    }
+
+    @Test public void  testRollBetweenOneAndSix(){
```

Comment:
You coded the dice to be between 0-6 but were testing for 0-4 which would cause the test to occasionally fail.

@Test public void  testRollBetweenOneAndSix() {
        int[] result = Library.roll(3);
        for (int number : result) {
            boolean actual = number > 0 && number < 6;
            assertTrue(actual);
        }
    }